src/imu_vn_100.cpp

```diff
@@ -484,7 +515,7 @@ void ImuVn100::Stream(bool async) {
       std::list<std::string> sgrp5 = {"BG5_YPR_U"};
       if (imu_compensated_) {
         grp1 |=  BG1_ACCEL | BG1_ANGULAR_RATE;
-        sgrp1.push_back("BG11_ACCEL");
+        sgrp1.push_back("BG1_ACCEL");
```

Comment:
Just to be clear; this is just a minor bugfix for what a debug print, correct?  This has no bearing on the actual functioning of the IMU.

Reply:
Yes! Came across it while making this patch. We can split it up into its own PR if you wish.

Reply:
Nah, that's fine.  Just making sure I understood.